pages/content/amp-dev/documentation/index-2021.html

```diff
@@ -0,0 +1,137 @@
+---
+$title: Get Started
+$view: /views/2021/base.html
+
+frameworks:
+  - name: Next.js
+    headline: Next.js
+    logo:
+      src: /static/img/documentation/nextjs.png
+      width: 105
+      height: 64
+    text: "Turn any React page into an AMP page."
+    url: https://nextjs.org/docs/advanced-features/amp-support/introduction
+  - name: WordPress
+    headline: WordPress
+    logo:
+      src: /static/img/documentation/wordpress.png
+      width: 64
+      height: 64
+    text: "All the AMP benefits in the WordPress CMS."
+    url: https://amp-wp.org/documentation/success-with-wordpress/
+  - name: Eleventy
+    headline: 11ty
+    logo:
```

Comment:
Can we use SVG icons for those?

Reply:
Ferdi does not have them at this point, but once I have the SVGs, I'm sure we can use those instead.

Reply:
PTAL